entc/load/schema_test.go

```diff
@@ -252,6 +252,10 @@ type WithDefaults struct {
 }
 
 func (WithDefaults) Fields() []ent.Field {
+	idf := field.Int("int_default_func")
+	idf.Descriptor().Default = func() int {
+		return 1e9
+	}
```

Comment:
Why not using the `DefaultFunc` option?

Reply:
added this as a regression test for the actual issue before actually implementing `DefaultFunc` - fixing now